### PR TITLE
Full website spelling check + some url improvements

### PR DESCRIPTION
**Pull request by bedroesb**
I went over the pages and fixed the words that my spelling checker didn't like ;)

As described in our markdown cheat sheet it is better to not include our own domain `https://rdmkit.elixir-europe.org/` when linking to internal pages.

### CODE REVIEW

**Comment by bedroesb** [comment]
@martin-nc If you would have time, feel free to go over the corrections I made :) or other people from @elixir-europe/rdmkit-editors!

File: pages/your_tasks/data_analysis.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Conda works by making a nested folder containing the traditional UNIX directory
 * Tips and tricks to navigate the landscape of software package management solutions:
     * If you need multiple tools/programming languages, but your machines have different OS types or versions, list packages in a Conda `environment.yml`.
     * If you need conflicting versions of some tools/libraries for different operations, make separate Conda environments.
-    * If you need a few open source libraries for my Python script, none which require complilation, make a `requirements.txt` and reference `pip` packages.
+    * If you need a few open source libraries for your Python script, none which require compiling, make a `requirements.txt` and reference `pip` packages.
```
Comment (martin-nc) [inline pages/your_tasks/data_analysis.md:74]: "...none which..." should be "...none of which...'

**Review by martin-nc — APPROVED** [review]
Great work @bedroesb! I just made a couple of comments on the text.

File: pages/your_tasks/data_analysis.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ Conda works by making a nested folder containing the traditional UNIX directory
 * Tips and tricks to navigate the landscape of software package management solutions:
     * If you need multiple tools/programming languages, but your machines have different OS types or versions, list packages in a Conda `environment.yml`.
```
Comment (Nazeeefa) [inline pages/your_tasks/data_analysis.md:72]: Suggested revision: _"If you need multiple tools or programming languages for a machine that has a different OS type or version, you can list relevant packages in Conda environment via your `environment.yaml` file."_ ?

If you want to tell a reader about fetching a list of packages in conda environment, you can mention `conda list -n <my_env>`.

File: pages/your_tasks/identifiers.md [hunk 3]
```diff
@@ -65,11 +65,11 @@ If the research institute or group has a centralized and structured system (such
...
 
 ### Considerations
 
-PIDs are essential to make your digitale object (datasets or resources) citable, enabling you to claim and receive credit for your research output. In turn, when you reuse someone else research output, you have to cite it.
+PIDs are essential to make your digital object (datasets or resources) citable, enabling you to claim and receive credit for your research output. In turn, when you reuse someone else research output, you have to cite it.
 
 There are a couple of different ways by which you can obtain a globally unique persistent identifier, and you need to decide which one is the best solution for your dataset or resource.
```
Comment (Nazeeefa) [inline pages/your_tasks/identifiers.md:74]: Suggested revision: _"There are different ways to obtain a globally unique persistent identifier, and you..."_

**Review by Nazeeefa — COMMENTED** [review]
Thank you @bedroesb 
Hope you find my review/comments helpful.

File: pages/your_tasks/data_analysis.md (hunk 2 above)
Comment (bedroesb) [reply pages/your_tasks/data_analysis.md:72]: I like more the use of "Manage the software you need in an OS independent way by listing all relevant packages in the Conda environment via your environment.yaml file.

What do you think?

File: pages/your_tasks/sensitive_data.md [hunk 4]
```diff
@@ -45,7 +45,7 @@ What is considered sensitive information is usually regulated by national laws a
...
 * Information in Life Science projects are for the most part categorised under health and genetic data and are considered special category data under the GDPR.
-* You need to assess whether or not your dataset contains personally identifying attributes. Note that combinations of attributes that are themselves not identifyable can be identifyable together.
+* You need to assess whether or not your dataset contains personally identifying attributes. Note that combinations of attributes that are themselves not identifiable can be identifiable together.
```
Comment (bedroesb) [inline pages/your_tasks/sensitive_data.md:48]:
```suggestion
* You need to assess whether or not your dataset contains attributes that can lead to the identification of a person. Note that combinations of attributes that are themselves not identifiable can be identifiable together. See the definitions described in the [How can you de-identify your data](#how-can-you-de-identify-your-data) section.
```

Comment (bedroesb) [reply pages/your_tasks/sensitive_data.md:48]: @Nazeeefa What do you think of this? The original sentence was not clearly written imo. "personally identifying attributes" is I think not correct English.

File: pages/your_tasks/identifiers.md (hunk 3 above)
Comment (bedroesb) [inline pages/your_tasks/identifiers.md:74]:
```suggestion
There are different ways to obtain a globally unique persistent identifier, and you need to decide which one is the best solution for your dataset or resource.
```

File: pages/your_tasks/data_analysis.md (hunk 2 above)
Comment (bedroesb) [inline pages/your_tasks/data_analysis.md:72]:
```suggestion
    * Manage the software you need in an OS-independent way by listing all relevant packages in your Conda environment via the `environment.yaml` file.
```

File: pages/your_tasks/data_publication.md [hunk 5]
```diff
@@ -28,7 +28,7 @@ Sometimes it is difficult to determine if publishing data you have at hand is th
 
 ### Solutions
 * If ethical, legal or contractual issues apply to your data (e.g. personal or sensitive data, confidential or third-party data, data with copyright, data with potential economic or commercial value, intellectual property or IP data, etc) ask help to the Legal Team, Tech Transfer Office or Data Protection Officer of your institute.
```
Comment (bedroesb) [inline pages/your_tasks/data_publication.md:30]:
```suggestion
* If ethical, legal or contractual issues apply to your data type (e.g. personal or sensitive data, confidential or third-party data, data with copyright, data with potential economic or commercial value, intellectual property or IP data, etc), ask for help and advice from the Legal Team, Tech Transfer Office, and/or Data Protection Officer of your institute.
```

File: pages/your_tasks/data_analysis.md (hunk 2 above)
Comment (Nazeeefa) [reply pages/your_tasks/data_analysis.md:72]: Sounds good!

File: pages/your_tasks/sensitive_data.md (hunk 4 above)
Comment (Nazeeefa) [reply pages/your_tasks/sensitive_data.md:48]: Sounds better! Thanks @bedroesb

File: pages/your_tasks/data_analysis.md (hunk 1 above)
Comment (bedroesb) [reply pages/your_tasks/data_analysis.md]: Done, thanks!

File: pages/your_domain/proteomics.md [hunk 6]
```diff
@@ -38,7 +38,7 @@ The Human Proteome Organisation (HUPO) Proteomics Standards Initiative ([HUPO-PS
 - The Minimum Information About a Proteomics Experiment (MIAPE) guidelines document.
 - mzML  - a standard format for encoding raw mass spectrometer output.
 - mzIdentML - a standard exchange format for peptides and proteins identified from mass spectra.
-- mzQuantML - a standard format that is intended to store the systematic description of workflows quantifying molecules (principly peptides and proteins) by mass spectrometry.
+- mzQuantML - a standard format that is intended to store the systematic description of workflows quantifying molecules (in principle peptides and proteins) by mass spectrometry.
```
[the remark this answers is not in the document]
Comment (bedroesb) [reply pages/your_domain/proteomics.md]: Done!

File: pages/your_tasks/identifiers.md [hunk 7]
```diff
@@ -49,7 +49,7 @@ If the research institute or group has a centralized and structured system (such
 * Do not use problematic characters and patterns into your local identifier ([Lesson 5. Avoid embedding meaning or relying on it for uniqueness](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2001414#pbio-2001414-g001)). Problematic strings can be misinterpreted by some software. In this case it is better to fix the bugs or explicitly declare this possible issue in documentation.
```
[the remark this answers is not in the document]
Comment (bedroesb) [reply pages/your_tasks/identifiers.md:49]: I personally like to use numbers when it is sequential, and the order matters...

[the remark this answers is not in the document]
Comment (bedroesb) [reply pages/your_tasks/identifiers.md:49]: I mean, I don't see how being it numbered is different from bullet points in this case :)

[the remark this answers is not in the document]
Comment (Nazeeefa) [reply pages/your_tasks/identifiers.md:49]: Yes I agree and I understand. 
It's just that when I go through solutions, there is a lot of information so I thought numbering might help even if it doesn't correspond to numbering (amount) of points mentioned under considerations.

**Comment by bedroesb** [comment]
thanks @Nazeeefa for your in depth checking!

**Pull request merged**